docs/conf.py

```diff
@@ -144,3 +132,19 @@ def _get_project_meta() -> dict[str, str]:
 
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = True
+
+
+# Mocks
+def mock_autodoc_to_not_document_class_super_type_if_it_is_object() -> None:
+    from sphinx.ext import autodoc
+
+    class MockedClassDocumenter(autodoc.ClassDocumenter):
+        def add_line(self, line: str, source: str, *lineno: int) -> None:
+            if line == "   Bases: :py:class:`object`":
+                return
+            super().add_line(line, source, *lineno)
+
+    autodoc.ClassDocumenter = MockedClassDocumenter
```

Comment:
Mind explaining what this is doing in the docstring?

Reply:
Fixed [here](https://github.com/py-mine/mcstatus/compare/afb0702d1c9fda079879cb0fb2a0b1ef92fb54f7..f33d6f5e6c80269ef9072380bd2966a4ca44843e).